docs/source/user-guide/tasks/work-with-packages.rst

```diff
@@ -13,19 +44,136 @@ environment:
+     Collecting package metadata (current_repodata.json): ...working... done
+     Solving environment: ...working... done
+     ...
+     Executing transaction: ...working... done
+     Using Conda environment /Users/adefusco/Desktop/iris/envs/default.
+     Added packages to project file: hvplot=0.7, dask.
 
-   NOTE: Replace PATH with the actual path to your package.
+#. OPTIONAL: In a text editor, open ``anaconda-project.yml`` to
+   see the new packages listed in the packages section. To install the packages
+   run ``anaconda-project prepare``.
 
-   EXAMPLE: To add the packages Bokeh and pandas::
+*****************
+Removing packages
+*****************
 
-     $ anaconda-project add-packages bokeh=0.12 pandas
-     conda install: Using Anaconda Cloud api site https://api.anaconda.org
-     Using Conda environment /home/alice/mystuff/iris/envs/default.
-     Added packages to project file: bokeh=0.12, pandas.
+To remove packages from the ``packages:`` list run::
 
-#. OPTIONAL: In a text editor, open ``anaconda-project.yml`` to
-   see the new packages listed in the packages section.
+  anaconda-project remove-packages package1 package2
+
+NOTE: Replace ``package1`` and ``package2`` with the names of
+the packages that you want to include. You can specify as many
+packages as you want.
+
+EXAMPLE: To remove the package hvplot::
+
+  $ anaconda-project remove-packages hvplot
+  Using Conda environment /Users/adefusco/Desktop/testproj/envs/default.
+  Removed packages from project file: hvplot.
+
+******************
+Using pip packages
+******************
+
+In addition to adding and removing Conda packages as shown above Pip packages
+can be specified using the ``--pip`` flag::
+
+  anaconda-project add-packages --pip package1 package2
+  anaconda-project remove-packages --pip package1 package2
+
+NOTE: Replace ``package1`` and ``package2`` with the names of
+the packages that you want to include. You can specify as many
+packages as you want.
+
+EXAMPLE: To add the ``requests`` package to the default environment::
+
+  $ anaconda-project add-packages --pip requests
+  Collecting requests
+    Using cached requests-2.25.1-py2.py3-none-any.whl (61 kB)
+  Requirement already satisfied: certifi>=2017.4.17 in ./envs/default/lib/python3.8/site-packages (from requests) (2020.12.5)
+  Collecting idna<3,>=2.5
+    Using cached idna-2.10-py2.py3-none-any.whl (58 kB)
+  Collecting chardet<5,>=3.0.2
+    Using cached chardet-4.0.0-py2.py3-none-any.whl (178 kB)
+  Collecting urllib3<1.27,>=1.21.1
+    Using cached urllib3-1.26.4-py2.py3-none-any.whl (153 kB)
+  Installing collected packages: urllib3, idna, chardet, requests
+  Successfully installed chardet-4.0.0 idna-2.10 requests-2.25.1 urllib3-1.26.4
+  Using Conda environment /Users/adefusco/Desktop/testproj/envs/default.
+  Added packages to project file: requests.
+
+Optionally, you can edit the ``anaconda-project.yml`` file to add packages using
```

Comment:
```suggestion
Optionally, you can edit the ``anaconda-project.yml`` file to add or remove packages using
```

Reply:
I've restructured this page a bit to make it more clear